HISTORY.rst

```diff
@@ -1,6 +1,11 @@
 =======
 History
 =======
+0.101.0 (2019-3-25)
+------------------
+* Updated version of Trafaret to a non-beta version
+* Updated keras to 2.2.3 or greater
```

Comment:
I remember you discussing the backward compatibility issues in the Slack - if this will not be compatible with the previous version, then
1) This is more 0.200 then 0.101 :)
2) This requires mentioning in the changelog

Reply:
so– this shouldn't actually change anything about how customers use the tool. No legitimately breaking changes, but it simply won't have prediction consistency across past versions for ResNet50.

Not sure to handle that in open source. Do I need to upgrade to 0.200 if the model doesn't have prediction consistency?

Reply:
I don't think so, BUT it will be nice to mention this in a changelog :) so that there will be no surprises

Reply:
Good idea :)

Reply:
Done!